exec/java-exec/src/main/java/org/apache/drill/exec/ops/BaseOperatorContext.java

```diff
@@ -39,27 +39,29 @@
  * version of the operator context and the full production-time context
  * that includes network services.
  */
-
 public abstract class BaseOperatorContext implements OperatorContext {
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(BaseOperatorContext.class);
 
-  protected final FragmentContextInterface context;
+  protected final FragmentContext context;
   protected final BufferAllocator allocator;
   protected final PhysicalOperator popConfig;
   protected final BufferManager manager;
+  protected OperatorStatReceiver statsWriter;
   private DrillFileSystem fs;
   private ControlsInjector injector;
 
-  public BaseOperatorContext(FragmentContextInterface context, BufferAllocator allocator,
-               PhysicalOperator popConfig) {
+  public BaseOperatorContext(FragmentContext context, BufferAllocator allocator,
+                             PhysicalOperator popConfig,
+                             OperatorStatReceiver stats) {
     this.context = context;
     this.allocator = allocator;
     this.popConfig = popConfig;
     this.manager = new BufferManagerImpl(allocator);
+    this.statsWriter = stats;
```

Comment:
This member was removed in a recent commit. Since the test and production versions need different implementation, the derived classes manage the operator stats.